client/src/pages/Dashboard.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PersonalModalButton from './PersonalModalButton';
+import './dashboard.css';
+
+
+export default function Dashboard() {

```

Comment:
Here, make sure to redirect the user if there is no user logged in

Reply:
ok